### ISSUE TITLE
Compiling on FreeBSD 10.3

### ISSUE THREAD

**Issue by computerfr33k** [grke/burp#449]
I am trying to compile in a FreeNAS jail.
The issue is I don't know how to compile it properly on FreeBSD since I don't know which packages are needed on BSD and what they're names are because I'm usually on Linux. I tried compiling it manually after using the ports tree because I needed to disable ipv6 in order to get it listening on ipv4 only in a jail, but now I get the following SSL issue.
```bash
2016-08-01 16:02:31: burp[10082] SSL connect error
140175405651640:error:1409441B:SSL routines:ssl3_read_bytes:tlsv1 alert decrypt error:s3_pkt.c:1472:SSL alert number 51
2016-08-01 16:02:31: burp[10082] after client
```

**Comment by grke** [comment]
Hello,
What burp version are you using?
In 2.0.x, I have a FreeBSD test machine that runs all the burp tests on every patch that I submit, and it has been fine. I don't run it in a jail though.

**Comment by computerfr33k** [comment]
I would like to use version 1.4.40 since I know it's stable and works, and I have had some problems with v2 in the past.
But I could test v2 again and see how it goes before switching my current server over.

**Comment by computerfr33k** [comment]
Actually I managed to figure out a way to get it to work without having to worry about needing to know what dependencies are needed. I just assumed the Makefile was the makefile from this project and didn't think to look in it for possibly specifying arguments for compilation to disable ipv6. But I noticed that there was already an option defined in the makefile for specifying the arguments passed to the configure script. So, I was able to do it that way. It is still weird that on the freshports site it says to use make config to change the options, but it doesn't actually do anything. But I am glad I managed to figure it out one way or another.
